e2e/app-template/template/src/constants.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+const CUSTOM_TRANSFER_TOKEN_ACCESS_KEY =
```

Comment:
This looks like a secret but really isn't. It's only a randomly generated string used for testing purpose :slightly_smiling_face:

Reply:
Can we make this a code comment? :)

Reply:
Making it all zeros would work too and make it obvious it's a test value

Reply:
We can do that indeed

Reply:
I'll update that tomorrow

Reply:
Done :slightly_smiling_face: